code-studio/src/js/header.js

```diff
@@ -129,31 +134,12 @@ header.build = function (stageData, progressData, currentLevelId, scriptName, pu
   }
 
   var popupLoaded = false;
-  function lazyLoadPopup(trophiesClicked) {
+  function lazyLoadPopup() {
     if (!popupLoaded) {
       popupLoaded = true;
-      $.ajax({
-        url: "/popup/stats",
-        data: {
-          script_id: stageData.script_id,
-          current_level_id: currentLevelId,
-          user_id: clientState.queryParams('user_id'),
-          section_id: clientState.queryParams('section_id'),
-          puzzle_page: puzzlePage
-        }, success: function (result) {
-          $('.header_popup_body').html(result);
-          if (trophiesClicked) {
-            jumpToTrophies();
-          }
-        }
-      });
-    } else if (trophiesClicked) {
-      jumpToTrophies();
+      $.getJSON(`/api/script_structure/${scriptName}`, progress.renderCourseProgress);
```

Comment:
Currently we rely on `.user_stats_block` to inherit a number of CSS styles for the component.  When I refactor the components to match the mocks I'll remove this dependency.